packages/node_modules/@ciscospark/plugin-lyra/src/occupants.js

```diff
@@ -0,0 +1,178 @@
+/**!
+ *
+ * Copyright (c) 2015-2017 Cisco Systems, Inc. See LICENSE file.
+ * @private
+ */
+
+import {SparkPlugin} from '@ciscospark/spark-core';
+import querystring from 'querystring';
+import {base64} from '@ciscospark/common';
+
+/**
+ * @class
+ * @extends {Lyra}
+ * @memberof Lyra
+ */
+const Occupants = SparkPlugin.extend({
+  namespace: `Lyra`,
+
+  /**
+   * Joins a lyra space, update every 10 minutes to keep alive for MANUAL
+   * @param {Types~LyraSpace} space
+   * @param {string} space.url
+   * @param {object} options
+   * @param {string} options.passType
+   * @returns {Promise}
+   */
+  join(space, options = {passType: `MANUAL`}) {
```

Comment:
@ianwremmel, the join is basically the user adds himself to the space (when a user manually adds a Spark Board from the web client). I think it makes more sense to call `spark.lyra.occupants.join(space)` than `spark.lyra.spaces.join(space)`.